pkg/scheduler/factory/factory.go

```diff
@@ -824,8 +824,12 @@ func (c *configFactory) invalidateCachedPredicatesOnDeletePod(pod *v1.Pod) {
 	if c.enableEquivalenceClassCache {
 		// part of this case is the same as pod add.
 		c.equivalencePodCache.InvalidateCachedPredicateItemForPodAdd(pod, pod.Spec.NodeName)
-		// MatchInterPodAffinity need to be reconsidered for this node,
+		// When a pod is deleted, existing inter pod affinity may become invalid.
+		// (e.g. this pod was preferred by some else, or vice versa).
+		//
+		// Thus, MatchInterPodAffinity need to be reconsidered for this node,
 		// as well as all nodes in its same failure domain.
+		//
 		// TODO(resouer) can we just do this for nodes in the same failure domain
 		c.equivalencePodCache.InvalidatePredicates(
```

Comment:
Do we need this? As Pod affinity is skipped.

Reply:
Oh, should be rebase problem, just updated. Thanks!